mlflow/cli.py

```diff
@@ -455,12 +454,26 @@ def gc(backend_store_uri, run_ids):
 
 cli.add_command(mlflow.models.cli.commands)
```

Comment:
Re: https://github.com/mlflow/mlflow/pull/3972/files#r560013572 - can we make `mlflow.models.cli` a conditional import here as well?